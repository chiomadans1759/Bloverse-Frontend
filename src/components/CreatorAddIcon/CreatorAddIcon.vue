<template>
  <main id="bl-creator-add">
    <div class="button-container text-center">
      <i class="fas fa-plus icon"></i>
    </div>
  </main>
</template>

<script>
export default {
  
}
</script>

<style lang="scss" scoped>
@import './CreatorAddIcon.scss';
</style>
