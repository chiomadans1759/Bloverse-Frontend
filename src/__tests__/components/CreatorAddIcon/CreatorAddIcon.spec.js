import { mount } from '@vue/test-utils'
import CreatorAddIcon from '@/components/CreatorAddIcon/CreatorAddIcon.vue';

describe('CreatorAddIcon Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(CreatorAddIcon , {});
    expect(wrapper.element).toMatchSnapshot();
  });
});
