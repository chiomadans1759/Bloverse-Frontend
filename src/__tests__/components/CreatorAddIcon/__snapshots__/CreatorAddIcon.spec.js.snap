// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatorAddIcon Component should render correctly 1`] = `
<main
  id="bl-creator-add"
>
  <div
    class="button-container text-center"
  >
    <i
      class="fas fa-plus icon"
    />
  </div>
</main>
`;
